lib/lib.error.sh

```diff
@@ -20,7 +20,7 @@ logmsg ()
 
 	msglevel="$1"; shift
 	msec=`date '+%N'`
-	stamp="[`date '+%b %d %T'`.`printf "%.3s" $msec`] $PROGNAME[$$]:"
+	stamp="[`date '+%b %d %T'`.`printf "%.3s" $msec`] ${PROGNAME}[$$]:"
```

Comment:
@eldering I think this is more correct but I'm not 100% sure if this has no unwanted side effects?

Reply:
No, that's fine. Actually less likely to have unwanted side effects, since it's clearer what the variable name is.